src/custom/components/Header/HeaderMod.tsx

```diff
@@ -200,6 +166,15 @@ export const UniIcon = styled.div`
   }
 `
 
+export const LogoImage = styled.img.attrs(props => ({
```

Comment:
Creating a styled component, which nows pulls the image attributes from the ThemeProvider.

Reply:
Also, we shouldn't add things to this @W3stside and I can help, but this should go somewhere else. 

I think is best if we do a PR on top of this, just cleaning up these things.